src/components/form/templates/common.ts

```diff
@@ -56,10 +56,15 @@ export function findTitle(data: any, fieldSchema: any, formSchema: any) {
     data = sortDataByProperties(data, subSchema.properties);
 
     const firstEntry = findFirstEntry(data, subSchema);
+
```

Comment:
I removed this. I think it's unnecessary, but if you really want it, the change belongs in a separate commit 🙂